controllers/grid/users/reviewer/form/ReviewerForm.inc.php

```diff
@@ -241,12 +242,16 @@ function fetch($request) {
 			'reviewDueDate' => __('reviewer.submission.reviewDueDate'),
 			'submissionReviewUrl' => __('common.url'),
 			'reviewerUserName' => __('user.username'),
+			'contextName' => $context->getLocalizedName(),
+			'contextUrl' => __('common.url'),
+			'editorialContactSignature' => strip_tags($user->getContactSignature(), "<br>"),
+			'submissionTitle' => strip_tags($submission->getLocalizedTitle()),
```

Comment:
I suspect, since we're now supporting HTML emails, we'll have to run many of our variables through htmlspecialchars. Consider e.g. a `<` in the article title. Article titles shouldn't support HTML (at least not until we have a single-line rich editor).